network-mapper/templates/mapper-deployment.yaml

```diff
@@ -25,6 +25,16 @@ spec:
           env:
             - name: OTTERIZE_DEBUG
               value: {{ .Values.debug | quote }}
+            - name: OTTERIZE_CLOUD_API_ADDRESS
+              value: {{ .Values.mapper.cloudApiAddress | "https://app.otterize.com/api" }}
+            - name: OTTERIZE_CLIENT_ID
+              value: {{ .Values.mapper.clientId | quote }}
+            - name: OTTERIZE_CLIENT_SECRET
+              value: {{ .Values.mapper.clientSecret | quote }}
+            - name: OTTERIZE_UPLOAD_INTERVAL_SECONDS
+              value: {{ .Values.mapper.uploadIntervalSeconds | "60" }}
+            - name: OTTERIZE_CLOUD_GRAPHQL_ENDPOINT
```

Comment:
I didn't notice that this was separate until I saw it in the docs - please put it into the API_ADDRESS variable so it's the same as the CLI and the intents operator.